File: actia_app/BachendCrud-master/BachendCrud-master/src/main/java/com/RHActia/actia_app/model/Employee.java
package com.RHActia.actia_app.model;

import jakarta.persistence.*;
import lombok.*;




@Getter
@Setter
@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Employee {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String firstname;
    private String lastname;
    private String email;
    private String gender;
    private String photo;


 }

File: actia_app/employee-managment-system/src/app/model/employee.ts
export class Employee {
    id: number = 0;
    firstname: string = '';
    lastname: string = '';
    email: string = '';
    gender: Gender = Gender.MALE;
    photo: string = '';
}

export enum Gender {
    MALE = 'male',
    FEMALE = 'female',
    OTHER = 'other'
}


